styles.css

```diff
@@ -1,6 +1,6 @@
 
 body {
-  width: 100vw;
+  /* overflow: hidden; */
```

Comment:
- [ ] to keep your code clean and professional, kindly get rid of any unnecessary comments /  unused code in your CSS file 😄

Reply:
Except for the first one, all comments that I have in my code I'm gonna use in the future.
(handy ones for fast toggling during the development process; or a reminder to check something).  <br>
Another mandatory review just for deleting one line, really?  <br>
But about button effects, thanks for the reminder. I simply forgot about them.

Reply:
@Wh1teCl0ud sir keeping your code clean is part of HTML/CSS best practices , wich is part of this project as well , so kindly remove any unused code and if you need them in the future put them back later , you can check this [thread ](https://questions.microverse.org/t/is-writing-down-comments-on-the-code-a-good-practice/687/2) if you have any questions ^^ goodluck